dask_cuda/utils.py

```diff
@@ -42,6 +43,11 @@ def setup(self, worker=None):
         if self.nbytes is not None or self.managed_memory is True:
             import rmm
 
+            if self.pool_async is True:
+                rmm.mr.set_current_device_resource(
+                    rmm.mr.PoolMemoryResource(rmm.mr.CudaAsyncMemoryResource())
+                )
```

Comment:
Does this break down if RMM managed memory is enabled? If so, I can check to make sure it's disabled.

Reply:
These are two different malloc calls, `cudaMallocAsync` and `cudaMallocManaged`, so I believe we can't have an async managed malloc.

@shwina could you confirm if my comment above is correct? Are we only able to use one or the other?

Reply:
Correct, there's no way to use both.

Reply:
Good to know! As resolved below, we now raise a `ValueError` if both are enabled.